tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -23,6 +23,8 @@ protected function setUp()
             return;
         }
 
+        $this->resetSharedConn();
```

Comment:
This is needed since a new functional test is added prior to this one. In this case, the platform instance associated with the shared connection has already initialized the type mappings and won't reinitialize them with the newly added type.